src/basic-attention-token/BasicAttentionTokenLandingPage.tsx

```diff
@@ -2,65 +2,60 @@ import {
   useTrackMatomoEvent,
   useTrackMatomoPageView,
 } from "@/hooks/useTrackWithMatomo";
-import { useIsMobile } from "@/hooks/useIsMobile";
-import { Background } from "@/components/Background/Background";
-import { LandingPageAppBar } from "@/components/AppBar/LandingPageAppBar";
-import { Box, Button, Link, Stack, Toolbar, Typography } from "@mui/material";
+import { Box, Button, Link, Stack, Typography } from "@mui/material";
 import { Link as RouterLink } from "react-router-dom";
-import notification from "@/assets/images/newsfeed.png";
-import mobileAd from "@/assets/images/news-mobile.png";
-import bat from "@/assets/images/basic-attention-token.svg";
 import { Trans } from "@lingui/macro";
+import basicattentiontoken from "@/assets/images/bat-logo-white.svg";
+import { LandingPageAppBar } from "@/components/AppBar/LandingPageAppBar";
+import { Background } from "@/components/Background/Background";
+import { useIsMobile } from "@/hooks/useIsMobile";
 
 export function BasicAttentionTokenLandingPage() {
-  useTrackMatomoPageView({ documentTitle: "BAT Landing Page" });
+  useTrackMatomoPageView({
+    documentTitle: "Basic Attention Token Landing Page",
+  });
   const isMobile = useIsMobile();
 
   return (
     <Background>
       <LandingPageAppBar />
-      <Toolbar sx={{ mb: 2 }} />
-      <Box display="flex" flexDirection="column" justifyContent="center">
-        <Stack
-          direction={{ md: "column", lg: "row" }}
-          spacing={3}
-          justifyContent="center"
-        >
-          <Stack
-            direction="column"
-            spacing={3}
-            maxWidth={750}
-            justifyContent="center"
+
+      <Box display="flex" justifyContent="center" mt={5}>
+        <Stack maxWidth={1000} spacing={3}>
+          <Box alignSelf="center">
+            {isMobile && <img src={basicattentiontoken} width="300px" />}
+            {!isMobile && <img src={basicattentiontoken} width="500px" />}
```

Comment:
nit: again use a media query, not in js.

You may need to write `<Box component="img" ... /> in order to get `sx` to be available.